cmd/kubectl-gadget/biolatency.go

```diff
@@ -106,6 +108,8 @@ func runBiolatencyStop(cmd *cobra.Command, args []string) error {
 	}
 	traceID := args[0]
 
+	utils.SigHandler(&traceID)
```

Comment:
Do we really need this handler in gadgets that have and explicit stop operation? I think in this case the user can just run `stop` again and this would be fine. What do you think?

Reply:
You are right, I think I will remove commit dedicated to `biolatency`.